frontend/src/libs/types/map.type.ts

```diff
@@ -0,0 +1,8 @@
+type MapType = {
```

Comment:
Not sure that we need that. So move all of that to the Map Component 
Also, it may cause application errors because there is no  google.maps until the google script is loaded